cmd/sudoku/main.go

```diff
@@ -0,0 +1,71 @@
+package main
+
+import (
+	"fmt"
+
+	"github.com/marioarranzr/sudoku"
+)
+
+var easySudoku = []int{
+	0, 0, 3, 0, 2, 0, 6, 0, 0,
+	9, 0, 0, 3, 0, 5, 0, 0, 1,
+	0, 0, 1, 8, 0, 6, 4, 0, 0,
+
+	0, 0, 8, 1, 0, 2, 9, 0, 0,
+	7, 0, 0, 0, 0, 0, 0, 0, 8,
+	0, 0, 6, 7, 0, 8, 2, 0, 0,
+
+	0, 0, 2, 6, 0, 9, 5, 0, 0,
+	8, 0, 0, 2, 0, 3, 0, 0, 9,
+	0, 0, 5, 0, 1, 0, 3, 0, 0,
+}
+
+var hardSudoku = []int{
+	6, 0, 0, 0, 0, 0, 1, 5, 0,
+	9, 5, 4, 7, 1, 0, 0, 8, 0,
+	0, 0, 0, 5, 0, 2, 6, 0, 0,
+
+	8, 0, 0, 0, 9, 4, 0, 0, 6,
+	0, 0, 3, 8, 0, 5, 4, 0, 0,
+	4, 0, 0, 3, 7, 0, 0, 0, 8,
+
+	0, 0, 6, 9, 0, 3, 0, 0, 0,
+	0, 2, 0, 0, 4, 7, 8, 9, 3,
+	0, 4, 9, 0, 0, 0, 0, 0, 5,
+}
+
+func main() {
+	var (
+		e, h []int
+		err  error
+	)
+	e, err = sudoku.Solve(easySudoku)
+	if err != nil {
+		fmt.Println(err)
+	}
+	h, err = sudoku.Solve(hardSudoku)
+	if err != nil {
+		fmt.Println(err)
+	}
+	fmt.Printf(
```

Comment:
this could have been separated out into a pretty print function

Reply:
and maybe wrapping it with "Easy Sudoku Solved?: %v\n"
to make it clear the print out is correct to the user. (err case works above for other cases so I know you can argue it's implied)